src/allmydata/test/test_system.py

```diff
@@ -40,20 +55,36 @@
     TEST_RSA_KEY_SIZE,
     SameProcessStreamEndpointAssigner,
 )
-from .common_web import do_http, Error
+from .common_web import do_http as do_http_bytes, Error
 from .web.common import (
     assert_soup_has_tag_with_attributes
 )
 
 # TODO: move this to common or common_util
 from allmydata.test.test_runner import RunBinTahoeMixin
 from . import common_util as testutil
-from .common_util import run_cli
+from .common_util import run_cli_unicode
 from ..scripts.common import (
     write_introducer,
 )
 
-LARGE_DATA = """
+def run_cli(*args, **kwargs):
+    """
+    Backwards compatible version so we don't have to change all the tests.
```

Comment:
Maybe make this backwards compat note secondary to some docs that say what this version actually does - which is operate on native strings, I guess?